src/shogun/features/CombinedFeatures.cpp

```diff
@@ -271,12 +270,12 @@ void CCombinedFeatures::remove_subset()
 
 	subset_changed_post();
 	SG_UNREF(processed);
-	SG_DEBUG("leaving %s::remove_subset()\n", get_name())
+	SG_DEBUG("leaving remove_subset()\n")
```

Comment:
I was thinking about the same thing for we already have the autogenerated context.